synfig-studio/src/gui/states/state_normal.cpp

```diff
@@ -46,6 +46,7 @@
 #include <synfig/angle.h>
 
 #include <synfigapp/main.h>
+#include <gtkmm-3.0/gtkmm/widget.h>
```

Comment:
Do we have to specify gtkmm version on include header now?

We already told the compiler we are using gtkmm-3.0 through build script (Autotools and CMake)
Specifying gtkmm version again on headers kinda look redundant

@ice0 @rodolforg

Reply:
Missed that 😉 
Thanks for checking, I’ll remove it.